table/server/common/pom.xml

```diff
@@ -56,5 +56,12 @@
       <version>${project.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
Follow the convention in other POM files:
```suggestion

    <!-- Internal test dependencies -->
    <dependency>
```

Reply:
also why is this dependency needed in this PR?

Reply:
`alluxio.ConfigurationRule` is in module `alluxio-core-common`. I need to add the dependency to import it.